script/Eth.deploy.s.sol

```diff
@@ -30,9 +34,9 @@ contract EthDeployer is Script {
             console.log("Skipping deployment because PRIVATE_KEY is not set");
             return;
         }
-        // check if balance is at least 2 * MINIMUM_CONNECTOR_DEPOSIT
-        if (address(deployerAddress).balance < 2 * Constants.MINIMUM_CONNECTOR_DEPOSIT) {
-            console.log("Skipping deployment because balance is less than 2 * MINIMUM_CONNECTOR_DEPOSIT");
+
+        if (address(deployerAddress).balance < (2 * REGISTRATION_FEE_ETH)) {
+            console.log("Skipping deployment because balance is less than 2 *  REGISTRATION_FEE_ETH");
             return;
```

Comment:
this return won't fail or stop deployment. It will proceed without `taraAddressOnEth` and `ethAddressOnTara` set